CesiumGeometry/include/CesiumGeometry/IntersectionTests.h

```diff
@@ -42,6 +42,38 @@ class CESIUMGEOMETRY_API IntersectionTests final {
       const glm::dvec2& triangleVertA,
       const glm::dvec2& triangleVertB,
       const glm::dvec2& triangleVertC) noexcept;
+
+  /**
+   * @brief Determines whether the point is completely inside the triangle.
+   *
+   * @param point The point to check.
+   * @param triangleVertA The first vertex of the triangle.
+   * @param triangleVertB The second vertex of the triangle.
+   * @param triangleVertC The third vertex of the triangle.
+   * @return Whether the point is within the triangle.
+   */
+  static bool pointInTriangle(
```

Comment:
This may be an opportunity to make naming more consistent?

Either... rename `pointInTriangle2d` to just `pointInTriangle`, since we are function overloading anyway.

Or... rename `pointInTriangle` to `pointInTriangle3d` to tag along with the precendent already set.